docs/recipes/equality.md

```diff
@@ -116,13 +116,13 @@ eqUserId.equals({ userId: 1, name: 'Giulio' }, { userId: 2, name: 'Giulio' }) //
 
```

Comment:
It would be good to rename `setoidArrayOfPoints` further up (lines 91–96) to, say, `arrayOfPoints` so it's less confusing now that the Setoid references are gone.

Reply:
good catch! I updated it to `eqArrayOfPoints`